writeengine/splitter/we_splitterapp.cpp

```diff
@@ -486,6 +486,10 @@ void WESplitterApp::invokeCpimport()
     {
         //we need something that works on Windows as well as linux
         v2[i++] = arg;
+        if (i >= v2.size())
```

Comment:
Could you pull this in a separate patch?

Reply:
https://github.com/mariadb-corporation/mariadb-columnstore-engine/pull/1847
https://github.com/mariadb-corporation/mariadb-columnstore-engine/pull/1848